client/src/feedbacker-components/comment/comment.test.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { mount } from 'enzyme'
+import configureMockStore from 'redux-mock-store'
+import { Provider } from 'react-redux'
+import Comment from './comment'
+
+let mockStore
+let testProps
+let middlewares
+
+const id = 123456
+const devComment = {
+  text: 'This doesn\'t work',
```

Comment:
`` instead no need to escape

Reply:
eslint: Strings must use single quote :)

Reply:
🥰